docker/packager/binary/build.sh

```diff
@@ -110,6 +110,15 @@ fi
 mv ./programs/clickhouse* /output
 [ -x ./programs/self-extracting/clickhouse ] && mv ./programs/self-extracting/clickhouse /output
 mv ./src/unit_tests_dbms /output ||: # may not exist for some binary builds
+mv ./utils/check-marks/check-marks /output ||: # may not exist for some binary builds
```

Comment:
Every one util should be added to https://github.com/ClickHouse/ClickHouse/blob/master/tests/ci/ci_config.py#L214-L238